src/utils.py

```diff
@@ -0,0 +1,7 @@
+import hikari
+from arc import GatewayClient
+
+async def get_guild(client: GatewayClient, event: hikari.GuildMessageCreateEvent):
+    return event.get_guild() or await client.rest.fetch_guild(event.guild_id)
+
+def role_mention(role: hikari.Snowflake | str): return f"<@&{role}>"
```

Comment:
I would personally put the return on a new line.
Since the role param is actually taking a role ID, I would call it `role_id` and also allow it to be an `int`.

Reply:
I am assuming `hikari.Snowflake` inherits the `int` type, but you reckon it should explicitly include `int` in the type?

Reply:
Oh yeah you're right, ignore my comment!